public/content/documentation/web/page-level/form.md

```diff
@@ -1,80 +1,598 @@
 ## General Notes
 
-How to test a form
+How to test a web form for accessibility.
 
 ## Condensed
 
 ### #a11y - Web Accessibility Acceptance Criteria
 
-How to test a header
+How to test a web form for accessibility
 
 1. Test keyboard only, then screen reader + keyboard actions
-
-   - Skip-links: Focus moves directly to the header or navigation
-
-   - Tab: Nothing, headings are not focusable unless they are actionable
-
-   - Arrow-keys: Headings are browsed
+   - Skip-links: Not commonly used for form entry, but useful for jumping to a form landmark.
+   - Tab: Focus moves to each form control in logical order, strong visual focus indicator.
+   - Forms mode: Enabled when screen reader lands on a form field.
```

Comment:
I do like the idea of including this, but since we don't generally refer to screen reader modes in the criteria, I would remove this from here and the mobile section